components/ui/sonner.jsx

```diff
@@ -0,0 +1,30 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { useTheme } from 'next-themes';
+// eslint-disable-next-line import/no-self-import
+import { Toaster as Sonner } from 'sonner';
+
+const Toaster = ({
+  ...props
+}) => {
+  const { theme = 'system' } = useTheme();
+
+  return (
+    (<Sonner
```

Comment:
The extra pair of `()` you are referring to is native in the `Toaster` from `sonner`, which comes like that from Shadcn-ui. I didn't make any changes to this component.

You can read more about this component here: [sonner](https://ui.shadcn.com/docs/components/sonner)

Reply:
I checked the code of the component from the manual installation section and I see only one pair of `()`. However it might be this way in the TS version and when installing the JS version it has an extra pair (prob typo).
If you say this is the code coming from shadcn/ui then it's ok like this